apps/common-app/src/examples/RuntimeTests/ReJest/RuntimeTestsApi.ts

```diff
@@ -4,10 +4,12 @@ import type { TestComponent } from './TestComponent';
 import type { SharedValue } from 'react-native-reanimated';
 import type { TestConfiguration, TestValue, NullableTestValue, BuildFunction } from './types';
 import { DescribeDecorator, TestDecorator } from './types';
+import { ValueRegistry } from './TestRunner/ValueRegistry';
 
 export { Presets } from './Presets';
 
 const testRunner = new TestRunner();
+const valueRegistry = new ValueRegistry();
```

Comment:
Can we make the `valueRegistry` a member of the `TestRunner` class? It is safer in terms of potentially handling memory cleanup, etc.